### PR TITLE
Bookstore: create and remove books

### PR DESCRIPTION
**Pull request by tiagorahal**
#### In this branch:
- Add books functionality.
- Remove books functionality.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.book];
+
+    case REMOVE_BOOK:
+      return state.filter((b) => b.id !== action.id);
+
+    default:
+      return state;
+  }
```
Comment (dasileker) [inline src/redux/books/books.js]: - [x] This function should use the hooks instead of state.

File: src/redux/books/books.js [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
+});
+
+export const removeBook = (id) => ({
+  type: REMOVE_BOOK,
+  id,
+});
+
+// side effects, only as applicable
+// e.g. thunks, epics, etc
+// export function getBook() {
```
Comment (dasileker) [inline src/redux/books/books.js]: - [x] Please keep your code clean.

File: src/components/books/booknew.js [hunk 3]
```diff
@@ -1,19 +1,55 @@
...
+
+import { v4 as uuidv4 } from 'uuid';
+
+import { addBook } from '../../redux/books/books';
+
+const BookNew = () => {
+  const dispatch = useDispatch();
+
+  const [title, setTitle] = useState('');
+  const [category, setCategory] = useState('');
```
Comment (dasileker) [inline src/components/books/booknew.js]: - [x] Also here use hooks instead of stateless function

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @tiagorahal** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


- Please make your PR move descriptive.

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**